src/registries/npm-registry.js

```diff
@@ -12,7 +12,7 @@ import {addSuffix, removePrefix} from '../util/misc';
 import isRequestToRegistry from './is-request-to-registry.js';
 
 const defaults = require('defaults');
-const userHome = require('user-home');
+const userHome = require('os').homedir();
```

Comment:
Being up at 4 AM to fix PR's is the highlight of my day, so I wouldn't bank on my best judgement.
It needs access to `isRootUser` (which is exported) and `getUid` (which is not).  Would you like me to make `getUid` an export of `src/constants.js`, or move both to `src/util`?